### DISCUSSION TITLE
Newbie: Doc correction and contribution questions

### DISCUSSION THREAD

**Discussion by topher1120** [open-telemetry/opentelemetry-java#3010] [Q&A]
Hi all, 
I'm pretty new to OpenTelemetry and it's been a very long time since I've contributed to open source.  I'm hoping to help out, especially in documentation for the near future and would appreciate help understanding the day-to-day processes so I can help effectively.

To that end, I've found a correction in the existing documentation around the `sdk-extensions/autoconfigure` submodule that I'm hoping to make as my first contribution.  Specifically, the description on the property otel.exporter.otlp.headers says multiple headers should be separated via _semicolons_.  My OpenTelemetry vendor uses headers to pass information related to my account when traces are collected.  In trying to configure this property, I realized that headers should actually be separated by _commas_.  I've traced the source code to verify.  (You can see usage in `SpanExporterConfiguration` and `MetricExporterConfiguration`.)

Here's the set of process questions.  I'm familiar with Git, but shaky on practical experience of PRs.  First, should I create an issue for this item before submitting a PR?  If I create an issue, how do I associate the PR with it?  What's the general rule for this kind of thing?

Finally, as I work through setting up my application to use OpenTelemetry, I see a few areas within the Java projects that were initially confusing and I would like to work on clarifying those pages.  Is there someone in the community more dedicated to documentation that I should chat with to understand the overall goals, style and plans for the documentation?

Thanks,
Chris

**Comment by jkwatson** [comment]
Hi @topher1120 . You don't need to create an issue for a documentation PR, unless you think that issue will probably end up encompassing more than a single PR. That being said, you are totally free to create the issue if you think it will help foster discussion before the changes are made. 

There is no one dedicated to documentation. We would love contributions that make the documentation easier to understand, navigate, etc. Thanks!

**Reply by topher1120** [reply]
Great, thanks for the help!  I'll submit the PR soon and I look forward to helping in the future!
